cmd/listFeed.go

```diff
@@ -16,6 +16,8 @@ var listFeedCmd = &cobra.Command{
 	Run:   listFeed,
 }
 
+var defaultTimeFormat = "2006-01-02 15:04:05.000 +0000 MST"
```

Comment:
As this line defines a package level variable, it's easy to miss given that it is in the `cmd/listFeed.go` file. Ideally this would be passed as a configuration parameter but doing so is outside the scope of this fix. A compromise would be to move this line to `cmd/root.go` so at least it is co-located with other configuration items.

Reply:
sure, I moved it to root.go file